platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -188,7 +189,10 @@ public static class ClassVersion {
      * Placeholder (since this is such a hotspot) to hold the results from {@link ConfigurationHolder#getConfigData(Class)}
      * rather than calling that method more than once during the lifecycle of a {@link VirtualRootNode} instance.
      */
-    private final VirtualMapConfig config = ConfigurationHolder.getConfigData(VirtualMapConfig.class);
+    private final Configuration configuration =
```

Comment:
This should be passes as an argument as well.

Reply:
Fixed in [this commit](https://github.com/hashgraph/hedera-services/pull/14695/commits/5b936699f0e81d172aeb5021b60e1969eb5feac4), aligning changes with https://github.com/hashgraph/hedera-services/pull/15562.